data-plane/config/channelv2/500-dispatcher.yaml

```diff
@@ -156,6 +160,9 @@ spec:
         - name: config-kafka-channel-data-plane
           configMap:
             name: config-kafka-channel-data-plane
+        - name: kafka-channel-channels-subscriptions
+          configMap:
+            name: kafka-channel-channels-subscriptions
```

Comment:
Because I found `DATA_PLANE_CONFIG_FILE_PATH` is at `/etc/channels-subscriptions/data` which is mounted at `kafka-channel-channels-subscriptions` .... @pierDipi